Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM elixir:1.10 AS dev
 RUN sed -i '/^mozilla\/DST_Root_CA_X3/s/^/!/' /etc/ca-certificates.conf && update-ca-certificates -f
 
 RUN apt -q update && \
-    apt -q install -y default-mysql-client inotify-tools festival && \
+    apt -q install -y default-mysql-client inotify-tools festival yarnpkg && \
```

Comment:
We're not using `yarn` in the development image (since we're running a `webpack` service in the `docker-compose.yml`). And we definitely not need it in the production one.